src/v2/Apps/Order/Components/SavedAddresses.tsx

```diff
@@ -103,7 +103,7 @@ const SavedAddresses: React.FC<SavedAddressesProps> = props => {
     const isDefaultAddress = address.node.isDefault
 
     return (
-      <BorderBox
+      <StackableBorderBox
```

Comment:
👍

Reply:
I started by manually setting the borders to fix the problem and when I was working to get the border-radius right it occurred to me that there should be a component for that 😅